content/applications/services/helpdesk/advanced/after_sales.rst

```diff
@@ -129,19 +159,26 @@ the :guilabel:`Reverse Transfer` pop-up window.
    :align: center
    :alt: View of a Helpdesk ticket with the return button highlighted.
 
-.. note::
+.. important::
    The :guilabel:`Return` button only appears on a ticket if the customer has a recorded delivery in
    the database.
 
-By default, the quantity will match the validated quantity from the delivery order. Update the
-:guilabel:`Quantity` field if necessary.
+Select a :guilabel:`Sales Order` or :guilabel:`Delivery to Return` to identify the products that
+need to be returned.
+
+By default, the quantity matches the validated quantity from the delivery order. Update the
+:guilabel:`Quantity` field if necessary. To remove a line, click the :guilabel:`🗑️ (trash can)`
+icon.
+
+Select a :guilabel:`Return Location` where the items should be directed after the return is
+completed.
```

Comment:
Not sure if you want, but perhaps a tip box can be created explaining that if the return is for a faulty product, it can go to a quarantined area/inspection area, etc. Just mention as a 'best practices' kind of thing. Totally optional!